firefly_widgets/_version.py

```diff
@@ -1,2 +1,5 @@
-version_info = (0, 1, '0-beta5')
-__version__ = '.'.join(map(str, version_info))
+version_info = (0, 1, 0, 'beta', 5)
```

Comment:
Let's change the version so we can make a new release for pip install. I suggest 'beta 6'